demo/image_demo.py

```diff
@@ -15,7 +18,12 @@ def parse_args():
     parser.add_argument('img', help='Image file')
     parser.add_argument('config', help='Config file')
     parser.add_argument('checkpoint', help='Checkpoint file')
-    parser.add_argument('--out-file', default=None, help='Path to output file')
+    parser.add_argument(
```

Comment:
need to update:
docs/en/user_guides/inference.md
docs/zh_cn/user_guides/inference.md
docs/en/user_guides/inference.md
docs/zh_cn/user_guides/inference.md